File: FigurasGeometricas/EntidadesCs/Rectangulo.cs
namespace EntidadesCs
{
	public class Rectangulo : Figura
	{
		public float Altura { get; set; }
		public float Base { get; set; }
		public Rectangulo(string nombre, float @base, float altura) : base(nombre)
		{
			Base = @base;
			Altura = altura;
		}
		public override float CalcularArea()
		{
			return Base * Altura;
		}
		public override string ToString()
		{
			return $"Soy {Nombre} y tengo la base de {Base} y la altura de {Altura}";
		}
	}
}

File: FigurasGeometricas/EntidadesVb/Rectangulo.vb
Public Class Rectangulo
	Inherits Figura
	Public Property Base As Single
	Public Property Altura As Single
	Public Sub New(nombre As String, base As Single, altura As Single)
		MyBase.New(nombre)
		Me.Base = base
		Me.Altura = altura
	End Sub
	Public Overrides Function CalcularArea() As Single
		Return Base * Altura
	End Function
	Public Overrides Function ToString() As String
		Return $"Soy {Nombre} y tengo la altura de {Altura} y la base de {Base}"
	End Function
End Class
